### PR TITLE
Misc wf fixes

### PR DESCRIPTION
**Pull request by TristanWright**

### CODE REVIEW

**Comment by TristanWright** [comment]
@cjh1  PTAL especially for #310

File: src/workflows/pyfr/common/steps/Visualization/Start/index.js [hunk 1]
```diff
@@ -55,11 +55,22 @@ const VisualizationStart = React.createClass({
...
+
+    if (this.state.serverType === 'Traditional') {
+      payload.cluster = { _id: this.state[this.state.serverType].profile };
+    } else if (this.state.serverType === 'EC2') {
+      payload.cluster = {
+        type: 'ec2',
+        machineType: this.state[this.state.serverType].machine,
+        clusterSize: this.state[this.state.serverType].clusterSize,
+        profileId: this.state[this.state.serverType]._id,
+      };
```
Comment (cjh1) [inline src/workflows/pyfr/common/steps/Visualization/Start/index.js]: Is there away we could avoid this duplicate code block, it appears in the PyFR workflow as well? We are also missing cluster name from the payload, which is a good reason to try a avoid the duplication is we can.

**Comment by cjh1** [comment]
LGTM resolve #316 for me.

**Comment by TristanWright** [comment]
excellent. merging...

**Pull request merged**